scripts/kokoro.sh

```diff
@@ -25,6 +25,7 @@ docker build -t grpc-web:ubuntu_16_04 \
 docker-compose build
 
 bazel test javascript/net/grpc/web/...
+bazel test net/grpc/gateway/examples/...
```

Comment:
Looks like this needs to be `bazel build` for now, instead of `bazel test`. Since there's no test target for now, the exit status of this command is 4, which invalidates the whole kokoro run.

Reply:
Done